AudioKit/Common/Internals/AudioUnit Host/AKAudioUnitManager.swift

```diff
@@ -137,7 +137,7 @@ open class AKAudioUnitManager: NSObject {
 
     // just get a non nil list of Audio Units
     private var linkedEffects: [AVAudioUnit] {
-        return _effectsChain.flatMap { $0 }
+        return _effectsChain.compactMap { $0 }
```

Comment:
I think this is too bleeding edge.

Reply:
it's just the current syntax, but ok

Reply:
hard to keep track with swift as they push you to upgrade all the time. I just try to stay on top of the changes now so I don't have all this refactoring to do.